### ISSUE TITLE
[WorldGuard] Confirmation on what works and doesn't?

### ISSUE THREAD

**Issue by ghost** [keepcalm/BukkitForge#427]
Haio, checked at the other issues but they all seem to report that one function or another works. Thing is.. None of them are working on mine. Could really use some confirmation here as to what works with what WG build / BF build.

Using WorldGuard 5.71 (Tried 5.7.0 as well, no change)
BF Build 198
MindCrack Pack 8.0.1 (FTB, haven't tested without FTB)
Loading works fine, no errors. In the console it even states that the things I disabled are disabled, but testing in-game everything seems enabled still. Even tried adding the config to each seperate dimension/world (how you wish to call it. End, nether, twilight forest, etc) but no luck.

WG Console log:
http://pastebin.com/19AAjZEN

WG Config:
http://pastebin.com/UVS4DeYp

To summarize
- Creepers damage blocks.
- Fireballs damage blocks.
- Enderman steal blocks.
- (All) Fire still spreads.
- TnT still damages blocks.
- Player / Mob still trample crops.
- Zombies break doors

Untested aka No idea:
- Anything with Wither.
- Anything with Enderdragon

So.. What build are you guys using of WG/BF and what functions work on that?

**Comment by EDSman11** [comment]
I can't seem to use the blacklist function of WorldGuard either...

**Comment by Mastergalen** [comment]
Those who are not members of a region are still able to build.
